bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gpi/Gpi.java

```diff
@@ -2031,6 +2032,7 @@ public ParamSet getParamSet(PioFactory factory) {
         }
 
         Pio.addParam(factory, paramSet, DISPERSER_PROP, getDisperser().name());
+        Pio.addParam(factory, paramSet, HALF_WAVE_PLATE_ANGLE_VALUE_PROP, String.valueOf(getHalfWavePlateAngle()));
```

Comment:
This seems fine for `ocs` code, but I wonder if we shouldn't use the [same mechanism](https://github.com/gemini-hlsw/ocs/blob/develop/bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/obscomp/SPInstObsComp.java#L40-L45) that is used to format the position angle (i.e., `NumberFormat`)?

Reply:
No, mostly just whether we should set it to two decimal places as we do for position angle.

Reply:
Ah I see. I think in practice we don't allow decimals but let me check